include/boost/next_prior.hpp

```diff
@@ -101,7 +103,7 @@ struct next_advance_impl< T, Distance, true >
 {
     static T call(T x, Distance n)
     {
-        std::advance(x, n);
+        boost::iterators::advance(x, n);
```

Comment:
Names in `boost` are mostly for backward compatibility, newer code should use `boost::iterators`.